oar/core/advisory_mgr.py

```diff
@@ -172,6 +172,22 @@ def change_advisory_status(self, status=AD_STATUS_REL_PREP):
         except Exception as e:
             raise AdvisoryException(f"change advisory status failed") from e
 
+    def get_requested_doc_security_ads(self,ads):
+        """
+        get requested  Docs and product security approval advisories
+        """
+        try:
+            doc_appr = []
+            security_appr = []
+            for ad in ads:
+                if ad.get_erratum_data()["doc_complete"] == 0:
+                    doc_appr.append(ad.errata_id)
+                if ad.get_erratum_data()["security_approved"] == False:
```

Comment:
is it possible that the value of `security_approved ` is null
```
advisory[security_approved]: null for security approval not requested, false for requested, true for approved
```

Reply:
if it's null or true, will nothing to do.

Reply:
a little confused here, `doc_complete` is not used to check doc approval is requested or not. 
```
advisory[doc_complete]: '1' to approve docs, '0' to disapprove docs
```
another question, why we get approval requested advisories here, we should get approved advisories.